pyccel/codegen/printing/fcode.py

```diff
@@ -296,10 +296,7 @@ def _print_Module(self, expr):
         sep = self._print(SeparatorComment(40))
         interfaces = ''
         if expr.interfaces:
-            interfaces = '\n'.join(self._print(i) for i in expr.interfaces if not i.hide)
-            for interface in expr.interfaces:
-                if not interface.hide:
-                    body += '\n'.join(''.join([sep, self._print(i), sep]) for i in interface.functions)
+            interfaces = '\n'.join(self._print(i) for i in expr.interfaces)
```

Comment:
We seem to have lost the insertion of the interfaces into the body

Reply:
I print the interfaces previous to the `contains` statement, that's why I have body and interfaces separated

Reply:
Found it! Thanks :+1: